cms/templates/course_outline.html

```diff
@@ -124,11 +124,11 @@ <h2 class="title title-3 warning-heading-text">${_("This course uses features th
         <div class="exam-settings-alert copy">
           <h2 class="title title-3">${_("This course has proctored exam settings that are incomplete or invalid.")}</h2>
           <p>
-            % if course_authoring_microfrontend_url:
+            % if mfe_proctored_exam_settings_url:
               <% url_encoded_course_id = quote(six.text_type(context_course.id).encode('utf-8'), safe='') %>
               ${Text(_("To update these settings go to the {link_start}Proctored Exam Settings page{link_end}.")).format(
-                link_start=HTML('<a href="{course_authoring_microfrontend_url}/proctored-exam-settings/{url_encoded_course_id}">').format(
-                  course_authoring_microfrontend_url=course_authoring_microfrontend_url,
+                link_start=HTML('<a href="{mfe_proctored_exam_settings_url}">').format(
```

Comment:
Why can't this be just
```
'<a href="{mfe_proctored_exam_settings_url}">'),
```

is there any particular reason to use the format here?